emulators/oberon-risc/Portfile

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+
+name                oberon-risc
+version             20200818
+categories          emulators
+platforms           darwin
+maintainers         {@jflude hotmail.com:justin_flude} openmaintainer
+description         Project Oberon RISC system emulator
+long_description    Project Oberon 2013, by Niklaus Wirth and Jürg Gutknecht, \
+    is a lean, easily understandable desktop computer. This emulation \
+    includes a disk image of the latest full system (note: make a scratch, \
+    writable copy)
+homepage            https://www.projectoberon.net/
+license             BSD
+
+checksums           rmd160 205aba103984e4606633f39cdb3d9b5a829e650c \
+                    sha256 43ca13736b0e5618a18d95efaeb57d529d71cb4f1fd43950103d25f24dfa05c9 \
+                    size 1073579
+
+PortGroup           github 1.0
```

Comment:
Usually this would be at the top; you might want to look at a couple of portfiles that use the portgroup to get a sense of how people usually lay it out.